lynx/src/main/java/com/github/pedrovgs/lynx/LynxView.java

```diff
@@ -219,7 +226,7 @@ private void initializeConfiguration(AttributeSet attrs) {
           attributes.getInteger(R.styleable.lynx_sampling_rate, lynxConfig.getSamplingRate());
 
       lynxConfig.setMaxNumberOfTracesToShow(maxTracesToShow)
-          .setFilter(filter)
+          .setFilter(TextUtils.isEmpty(filter)?"":filter)
```

Comment:
This line is breaking your build, you have to change this line to:

```java 

.setFilter(TextUtils.isEmpty(filter) ? "" : filter)

``` 

And you build will work again!!!